### PR TITLE
AMP for Email Addition

### PR DESCRIPTION
**Pull request by EmilyNecciai**
# Pull Request/Issue Resolution

**Description of Change:**
Adding info for AMP for Email (AMPHTML). 

**Reason for Change:**
In Beta, but need public docs for functionality.

### CODE REVIEW

File: _docs/_partners/technology_partners/channel_extensions/email_orchestration/amphtml.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  "should_inline_css": (optional, boolean), whether to inline css on the body. If not provided, falls back to the default css inlining value for the App Group,
+  "attachments": (optional, array), array of json objects like [{"file_name","url"}] that define the files you need attached. Your file name's extension will be detected automatically from the URL, which should return the appropriate `Content-Type` as a response header,
+}
+```
+
+## Writing Your AMP Email
+
+Construct your AMP Email using the [elements](#elements) below, then use [our API](#api-usage) to send. Be sure to use `amp_body` for your AMP HTML! You can also check out [AMP's tutorial](https://amp.dev/documentation/guides-and-tutorials/start/create_email?format=email) or [sample code](https://gist.github.com/CrystalOnScript/988c3f0a2eb406da27e9d9bf13a8bf73) to see how the final product should look.
+
+We highly recommend having a `plaintext_body` version, as well as require a regular HTML `body` version of your email, in the event that your email is sent to a provider who does not yet support AMP for Email.
```
Comment (gsinghAppboy) [inline _docs/_partners/technology_partners/channel_extensions/email_orchestration/amphtml.md]: the 'regular html body' is always required.  So we should re-phrase this line

File: _docs/_partners/technology_partners/channel_extensions/email_orchestration/amphtml.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  "extras": (optional, valid Key-Value Hash), extra hash - for SendGrid customers, this will be passed to SendGrid as Unique Arguments,
+  "headers": (optional, valid Key-Value Hash), hash of custom extensions headers. Currently, only supported for SendGrid customers,
+  "should_inline_css": (optional, boolean), whether to inline css on the body. If not provided, falls back to the default css inlining value for the App Group,
+  "attachments": (optional, array), array of json objects like [{"file_name","url"}] that define the files you need attached. Your file name's extension will be detected automatically from the URL, which should return the appropriate `Content-Type` as a response header,
+}
+```
+
+## Writing Your AMP Email
+
+Construct your AMP Email using the [elements](#elements) below, then use [our API](#api-usage) to send. Be sure to use `amp_body` for your AMP HTML! You can also check out [AMP's tutorial](https://amp.dev/documentation/guides-and-tutorials/start/create_email?format=email) or [sample code](https://gist.github.com/CrystalOnScript/988c3f0a2eb406da27e9d9bf13a8bf73) to see how the final product should look.
```
Comment (gsinghAppboy) [inline _docs/_partners/technology_partners/channel_extensions/email_orchestration/amphtml.md]: let's just remove the 'sample code' since it can be gotten from the amp.dev website.  Also not sure who's code that is, in the long run it could not be supported and cause us issues.

**Comment by EmilyNecciai** [comment]
> waiting for the deploy....not sure what the #elements stuff is. Left 2 comments for minor changes. Rest looks great!
> 
> Confirming, this is still behind a hidden page right?

Lot's of changes coming with this push, including the hidden tag. :) Should it be in partners or should it be elsewhere?

File: _docs/_hidden/private_betas/amphtml.md [hunk 3]
```diff
@@ -1,15 +1,36 @@
...
 
-## AMPHTML Email via API
+## Requirements
 
-When you use any of [our Messaging Endpoints]({{ site.baseurl }}/api/endpoints/messaging/) to send an email, add `amp_body` as an object specification, as shown below.
+Braze is not responsible for the customer registering with Braze or meeting the necessary security requirements.
+
+Requirement   | Description
+--------------| -----------
+AMP Account   | You must have an account with AMP for Email to utilize their programs with Braze.
```
Comment (gsinghAppboy) [inline _docs/_hidden/private_betas/amphtml.md]: not sure what this is referencing, there is not really a concept for AMP Account.  Maybe we should rephrase this to say that you need to have AMP enabled for your Gmail account, in order to view AMP HTML content.
